ui/app/store/actions.js

```diff
@@ -2125,6 +2126,23 @@ export function setUseNonceField (val) {
   }
 }
 
+export function setUsePhishDetect (val) {
+  return (dispatch) => {
+    dispatch(showLoadingIndication())
+    log.debug(`background.setUsePhishDetect`)
+    background.setUsePhishDetect(val, (err) => {
+      dispatch(hideLoadingIndication())
+      if (err) {
+        return dispatch(displayWarning(err.message))
+      }
+    })
+    dispatch({
+      type: actionConstants.SET_USE_PHISH_DETECT,
```

Comment:
What does this action do? It appears to be unused.

Reply:
removed